x-pack/plugins/security_solution/public/timelines/store/timeline/reducer.test.ts

```diff
@@ -31,7 +31,7 @@ import { defaultHeaders } from '../../../common/mock';
 
```

Comment:
nit, i wish this file was called `helpers.test.ts`

Reply:
@stephmilovic,  Thanks for the comment. Currently, I do not have opinion. I guess I will need to write more test to familiarize with this part of the code. So, for now I won't take any actions. But I will do keep it in mind.